src/api/cpp/cvc5.h

```diff
@@ -3647,7 +3694,7 @@ class CVC5_EXPORT Solver
   /* .................................................................... */
 
   /**
-   * Create (first-order) constant (0-arity function symbol).
+   * Create a free constant.
```

Comment:
Since if you pass a *function* sort as `sort`, it is not 0-ary.
Note that:
`(declare-fun f () (-> Int Int))`
is equivalent to:
`(declare-fun f (Int) Int)`

Reply:
The sort passed as an argument is the codomain sort, though?

Reply:
No. If `sort` is not a function-like sort, it has no codomain.  If `sort` is a function-like sort, then its codomain is the codomain of `sort`.